test/stripe/api_resource_test.rb

```diff
@@ -66,6 +66,19 @@ class NestedTestAPIResource < Stripe::APIResource
       end
     end
 
+    should "putting api_key as a param instead of an opt when listing objects should trigger a warning" do
+      stub_request(:get, "#{Stripe.api_base}/v1/customers").
+        to_return(body: JSON.generate(API_FIXTURES.fetch(:customer)))
+      old_stderr = $stderr
+      $stderr = StringIO.new
+      begin
+        Stripe::Customer.list({ :api_key => 'sk_test_local' })
+        assert_match Regexp.new('WARNING:'), $stderr.string
+      ensure
+        $stderr = old_stderr
+      end
+    end
```

Comment:
Let's get rid of this test too. I think just the two below give us enough coverage.